pyrobosim/pyrobosim/core/hallway.py

```diff
@@ -1,5 +1,7 @@
 """Hallway representation for world modeling."""
 
+from __future__ import annotations
```

Comment:
Why is this needed? For the `set[Hallway]` annotation? Can this instead be imported from `typing`?